src/seedu/addressbook/AddressBook.java

```diff
@@ -412,12 +433,12 @@ private static String getMessageForInvalidCommandInput(String userCommand, Strin
      * Adds a person (specified by the command args) to the address book.
      * The entire command arguments string is treated as a string representation of the person to add.
      *
-     * @param commandArgs full command args string from the user
+     * @param COMMAND_ARGS full command args string from the user
      * @return feedback display message for the operation result
      */
-    private static String executeAddPerson(String commandArgs) {
+    private static String executeAddPerson(String COMMAND_ARGS) {
```

Comment:
Again the method argument should not be in all caps as it is not and should not be a constant.